engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetWireframe.java

```diff
@@ -26,6 +26,9 @@
  * TODO: Add javadocs
  */
 public final class SetWireframe implements StateChange {
+    public static final int ENABLED = GL_LINE;
+    public static final int DISABLED = GL_FILL;
+
```

Comment:
I don't think the StateChange implementation should have OpenGL knowledge if we can avoid it. See below.